tyr-core/src/main/java/org/jboss/tyr/check/TemplateChecker.java

```diff
@@ -122,6 +125,13 @@ public String checkPR(JsonObject payload) throws InvalidPayloadException {
     private List<Check> registerChecks(Format format) {
         List<Check> checks = new ArrayList<>();
 
+        if (format.getLabel() != null) {
+            if (format.getLabel().getTargetBranch() != null) {
+                addLabelCheck.validateLabel(format.getLabel().getTargetBranch().get(0));
```

Comment:
That will always get the first branch definition. If it should be possible to define details for multiple branches, this should be a map.

Reply:
I do not think that this as map is possible without redoing whole addLabelCheck since it is very specific for getting the target branch information through GH API. I was thinking that if we want to add more labels, every label would have its own class. Otherwise, I agree to remove List<>.